gnome-extra/cinnamon/cinnamon-6.2.3.ebuild

```diff
@@ -95,8 +94,15 @@ RDEPEND="
 	x11-themes/adwaita-icon-theme
 	x11-themes/gnome-themes-standard
 
+	desktop-portal? (
+		sys-apps/xdg-desktop-portal-gtk
+		sys-apps/xdg-desktop-portal-xapp
+	)
+	gnome-online-accounts? (
+		>=net-libs/gnome-online-accounts-3.18.0:=
+	)
```

Comment:
The USE flag does nothing but add a dependency. It is the wrong dependency though. The settings module checks if https://github.com/xapp-project/gnome-online-accounts-gtk is installed and dynamically adds an entry.

I think it works better as optfeatures, personally, either way that needs to be packaged first.

Reply:
Yuck. I didn't realize that was referencing a different app entirely.

I don't use GOA, and I'm not exactly jumping at the prospect of maintaining another package (especially one I won't be using), so I'm going to leave this out for now. If there's enough demand for it, we can revisit.

Reply:
Yeah I don't use it either. I would just leave it out entirely then. ;)